samples/v1beta1/buildstrategy/multiarch-native-buildah/rolebinding_multiarch_native_buildah_cr.yaml

```diff
@@ -4,8 +4,8 @@ metadata:
   name: multiarch-native-buildah-pipeline
 roleRef:
   apiGroup: rbac.authorization.k8s.io
-  kind: Role
+  kind: ClusterRole
   name: multiarch-native-buildah-pipeline
 subjects:
   - kind: ServiceAccount
-    name: pipeline
+    name: ${SERVICE_ACCOUNT}
```

Comment:
This breaks the test. Looking into it.

Reply:
Added a `sed` command to replace the SA name & apply the file instead of using a variable inside the yaml.